sass/components/_ciq-form.scss

```diff
@@ -8,8 +8,8 @@
     .slider {
         width: 120px;
         height: 10px;
-        display: relative;
         cursor: pointer;
+        touch-action: manipulation;
```

Comment:
just added this to stop `Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive.` error message. Refer https://github.com/OwlCarousel2/OwlCarousel2/issues/1932